integration-tests/src/test/resources/log4j2-pretest.xml

```diff
@@ -2,21 +2,21 @@
 <Configuration packages="eu.tsystems.mms.tic.testframework.logging">
     <Appenders>
         <Console name="CONSOLE">
-            <PatternLayout pattern="%d{dd.MM.yyyy HH:mm:ss.SSS} [%t][%p]%contextIds: %c{2} - %m{nolookups}%n" />
```

Comment:
Sanity Check: Is it correct that this file was _moved_, not copied? I tried to check for references on master, but did not find any.